projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.scss

```diff
@@ -64,11 +64,13 @@
 }
 
 .notification-panel {
```

Comment:
While you're at it, please use our BEM style notation, e.g. `.m-location-picker__notification-panel`